<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tetkikduzenle</class>
 <widget class="QDialog" name="tetkikduzenle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetkik Düzenle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblAdi">
       <property name="text">
        <string>&amp;Tetkik Adı</string>
       </property>
       <property name="buddy">
        <cstring>leAdi</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leNormaldegerErkek">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leAdi">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblDegerler">
       <property name="text">
        <string>&amp;Normal Değerler Erkek</string>
       </property>
       <property name="buddy">
        <cstring>leNormaldegerErkek</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="teAciklama">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAciklama">
       <property name="text">
        <string>&amp;Açıklama</string>
       </property>
       <property name="buddy">
        <cstring>teAciklama</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblDegerler_2">
       <property name="text">
        <string>&amp;Normal Değerler Kadın</string>
       </property>
       <property name="buddy">
        <cstring>leNormaldegerKadin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leNormaldegerKadin"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnKaydet">
       <property name="text">
        <string>Kaydet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVazgec">
       <property name="text">
        <string>Vazgec</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
